docs/Makefile

```diff
@@ -17,5 +17,4 @@ help:
 # Catch-all target: route all unknown targets to Sphinx using the new
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
-	../examples/generate_notebooks.sh
```

Comment:
Would be neat if we could integrate the `generate_notebooks.py` into the makefile. 

e.g. through additional arguments. 

What do you think?

Reply:
I would leave Sphinx MAKEFILE untouched. 

If a script to fully build the docs is needed, should probably create a separate build documentation Python script;

Reply:
why?

Reply:
Easier to maintain than both Makefiles build scripts in Sphinx.

Reply:
but you would still have only one python script ... The makefiles just use it.
Or what is the overhead in maintenance if you call the script from within the makefiles? It would be much easier to build the documentation the same way the ci does it